std/stdio.d

```diff
@@ -3640,6 +3667,69 @@ void main()
     }
     assert(std.file.readText!string(deleteme).stripLeft("\uFEFF") == "foobar");
 }
+@safe unittest // char/wchar -> wchar_t
```

Comment:
Is there any reason for this unittest to not just be `@system` , what with all the `@trusted` lambdas?

Reply:
The thing that's actually being tested is `writer.put(s)`. And we might want to ensure that that's `@safe`. The `@trusted` parts are just setup for the test.

But truth be told, I just started with `@safe` and added `@trusted` as needed. I can change it to `@system` if you like that better.

Reply:
fair enough.